.env-example

```diff
@@ -3,4 +3,5 @@ DB_HOST=
 DB_PORT=
 DB_USER=
 DB_PASSWORD=
-DB_NAME=
\ No newline at end of file
+DB_NAME=
+SALTORROUNDS=
```

Comment:
Can you rename this environment variable follow naming convention rules?
Suggestion: SALT_ROUNDS